core/data_access/base_models/base_model.py

```diff
@@ -0,0 +1,37 @@
+from django.utils import timezone
+from django.db import models
+from core.data_access.base_models.model_event import ModelEvent
+
+
+class BaseModel(models.Model):
+    _events = []
+    created_at = models.DateTimeField(auto_now_add=True)
```

Comment:
This is not handled correctly:

{
  "success": false,
  "data": null,
  "error": "An unexpected error occuredcolumn \"created_at\" of relation \"player\" does not exist\nLINE 1: INSERT INTO \"player\" (\"created_at\", \"updated_at\", \"firstname...\n                              ^\n"
}

Reply:
We need 

created_at = models.DateTimeField(blank=True, null=True)
updated_at = models.DateTimeField(blank=True, null=True)

in player_model.py

Reply:
You need to migrate to add the columns. Columns should not be allowed null and will then be set automatically by BaseModel on .Save()